Sources/Logging/Logging.swift

```diff
@@ -603,6 +629,15 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
+#if canImport(WASILibc)
+// There is no stdio/FILE in WASI yet, neither stdout/err. But it does
+// support `print`.
```

Comment:
how does WASILibc's `Swift.print` actually work?

Reply:
I skimmed over the source but couldn't find a specific change, maybe the WASILibc just doesn't export stdio.h for some reason.

Reply:
`stdio`/`stderr` should be available if you add `import WASILibc`, most probably this `PrintOutputStream` won't be needed.